Composer/packages/client/src/pages/setting/deployment/deployWizardStep-createDeploy.js

```diff
@@ -23,6 +24,7 @@ import { styles } from './styles';
 import { regionOptions } from './luisRegions.js';
 
 export const DeployWizardStepCreate = props => {
+  const defaultRegion = 'WestUS';
```

Comment:
I don't think we should hard code here, we should assume the options[0] is default.

Reply:
Yeah, but I am not sure the other 2 regions is it OK for everyone. because currently our luis authoring accounts are mostly for westus region. Should we give a hint to user?

Reply:
removed other 2 regions, use options[[0] as default.